flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/MFD/A320_Neo_MFD.js

```diff
@@ -853,16 +853,21 @@ class A320_Neo_MFD_NDInfo extends NavSystemElement {
         }
         const ADIRSState = SimVar.GetSimVarValue("L:A320_Neo_ADIRS_STATE", "Enum");
         const groundSpeed = Math.round(Simplane.getGroundSpeed());
+        const trueAirSpeed = SimVar.GetSimVarValue("AIRSPEED TRUE", "knots");
+        const windStrength = SimVar.GetSimVarValue("AMBIENT WIND VELOCITY", "knots");
         const gs = this.ndInfo.querySelector("#GS_Value");
         const tas = this.ndInfo.querySelector("#TAS_Value");
         const wd = this.ndInfo.querySelector("#Wind_Direction");
         const ws = this.ndInfo.querySelector("#Wind_Strength");
         const wa = this.ndInfo.querySelector("#Wind_Arrow");
         const wptg = this.ndInfo.querySelector("#Waypoint_Group");
         const wptPPOS = this.ndInfo.querySelector("#Waypoint_PPOS");
-        if (ADIRSState != 2 || groundSpeed <= 100) {
-            //Hide TAS, and wind info
+        if (ADIRSState != 2 || trueAirSpeed < 60) {
```

Comment:
```suggestion
        if (ADIRSState !== 2 || trueAirSpeed < 60) {
```
can we use strict comparison?